app/(mainSite)/events/page.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { Suspense } from "react";
+
+import Footer from "../../../components/Footer/Footer";
+import EventCards from "../../../components/PageCard/EventCards";
+import { Box, Center, Grid, styled } from "../../../styled-system/jsx";
+import { css } from "../../../styled-system/css";
+
+// TODO: Update docs
```

Comment:
Can this be done in this PR or are you waiting for some additional information from Jonah?

Reply:
I was just trying to get this out ASAP mainly for the impactathon F.A.Q, I'll add the docs now.